sgpc3/sgpc3_example_usage.c

```diff
@@ -59,16 +59,11 @@ int main(void) {
      * restoring it after with sgp_get_iaq_baseline / sgp_set_iaq_baseline.
      * If a recent baseline is not available, reset it using
      * sgp_iaq_init_continuous prior to running IAQ measurements.  */
-    err = sgp_measure_signals_blocking_read(&scaled_ethanol_signal);
+    err = sgp_measure_signals_blocking_read(&ethanol_signal);
 
     if (err == STATUS_OK) {
         /* Print ethanol signal with floating point support */
```

Comment:
Comment about printing with floating point support is not needed anymore.

Reply:
done